feat: adding team 28 visualisation and gallery page for project 1.

rahulravindran0108 opened the pull request:
I have added a new page and made it similar to the visualisation tab. This page would link to gh-pages for Project 1. Please have a look and give a thumbs up/thumbs down (or merge directly) :)

rahulravindran0108 committed:
feat: adding team 28 visualisation and gallery page for project 1.

lorsposto commented:
Thanks for the contribution. Intuitively, these vizs would also be under the Visualizations link in the nav bar, but that section is still a works in progress regarding its organization so that can be something we shift later. 

There are also some cosmetic inconsistencies like MIME types link is only present in index.html, or USC branding on the viz pages. @rahulravindran0108 you can take that on if you'd like, but it's no big deal, something the team can do as well. 

:+1: I say merge. @luuj @ruthvik947 @janethkim @titusjung thoughts?

rahulravindran0108 committed:
fix: fixing link issues

rahulravindran0108 committed:
Revert "fix: fixing link issues"

This reverts commit 3b54d927392b9f267a3726cb6b617e75d124ca1f.

rahulravindran0108 commented:
@lorsposto seems like there were commits after my PR. Should I close and open a new pull ?

Update: fixed MC and updates PR.

rahulravindran0108 committed:
Merge commit '8bb5d261284ca37a053e6868ab758e72eec402d8' into gh-pages

Conflicts:
	index.html

rahulravindran0108 committed:
fix: fixing merge conflicts and link issues

chrismattmann commented:
+1 to merge, please also include the actual JSON data, and files so that it can be part of polar.usc.edu @rahulravindran0108 @lorsposto

rahulravindran0108 committed:
feat: pushing data foe team 28 for bfc, fht and bfa.

Divided output files into three parts in order to ensure I meet github limits.
also added heat map data for unknown files in octet-stream.

rahulravindran0108 commented:
@chrismattmann @lorsposto done, added team28 to data folder. Divided the output json into three parts for meeting github limits on file size.

chrismattmann commented:
thank you @rahulravindran0108 you rock! @lorsposto please add a credits main to the site, and add CS401 team, along with myself, and @rahulravindran0108 as well. We should also credit the students in that page from the 572 class as well. +1 to merge.

chrismattmann commented:
so happy great work again @rahulravindran0108

ruthvik947 commented:
Hello @rahulravindran0108! This looks wonderful thank you! A couple things, though:
- instead of having the source on your website (i.e. having the MIME analysis thumbnail link to your website) would you be able to add the actual d3 source to this repository? It might make sense to have all the visualizations in one consolidated location.
- instead of having the MIME analysis be a different navbar section, you could have it show up in the 'visualizations' page. I think this would add consistency to the site.

@lorsposto @luuj @chrismattmann let me know if this makes sense!

chrismattmann commented:
@ruthvik947 can you review the latest PR I believe it addresses these comments?

chrismattmann commented on html/mime-analysis.html:
```diff
@@ -0,0 +1,147 @@
+                            <div class="caption">
+                                <h3>MIME Analysis Team 28</h3>
+                                <p>These visualisations contribute to explaning the MIME Diversity in Polar Dataset.</p>
+                                <p><a href="http://rahulravindran0108.github.io/Polar-Dataset-Explorer/" class="btn btn-primary" role="button">View</a>
```
can you replace this external link with the actual visualizations and add them here to the pull request?

ruthvik947 commented:
@chrismattmann I pulled the PR to test - I believe it still links to rahul's website. @lorsposto @luuj can you confirm?

Edit: just saw your comment!

chrismattmann commented:
@ruthvik947 you are correct! just updated my comments thanks for the catch missed that link. cc @rahulravindran0108

rahulravindran0108 commented:
@ruthvik947 it still points to my website. I have a few questions w.r.t this issue
- I thought each project would have its own navigation system, so I decided to just point to gh-pages of the project. This would keep the design of the website consistent. Everyone will have their own navbar design, hence I decided to do this.
- However, if you would like to navigate to project html pages than each project files would need a backlink to polar website.

Either way, do let me know how to approach to this problem. I will add in the html files if need be.

rahulravindran0108 commented:
@chrismattmann Working on it. Adding back links to my project so that it points to USC polar. I Should be done in 30 minutes.

Edit: @ruthvik947 @lorsposto @chrismattmann Done.

rahulravindran0108 committed:
feat: adding html files for team 28

Also adding dependencies for js and css.
Updating project to include a back link to Polar Home Page.

chrismattmann commented:
thanks, to be clear @rahulravindran0108 I was wondering if you would contribute a fully isolated version of your project/site to polar.usc.edu so it doesn't contain backlinks other than to credit you on the main credits page for contributing. If you'd like we can make those point to your original site, but I want polar.usc.edu to not be dependent on external sites. Make sense?

rahulravindran0108 committed:
fix: fixed back links.

rahulravindran0108 commented:
@chrismattmann done.

chrismattmann commented:
thanks @rahulravindran0108

ruthvik947 commented:
Made a new PR - #20 
Thanks again @rahulravindran0108 !
